src/services/redis.ts

```diff
@@ -48,8 +48,9 @@ if (process.env.LOCAL === "docker-compose") {
 }
 
 export const redis = new Redis(connectionObj)
-export const redisSub = new Redis(connectionObj)
-export const pubsub = new RedisPubSub({
+
+const redisSub = new Redis(connectionObj)
```

Comment:
`redisSub` needed in `test/jest.setup.js` for tests

```suggestion
export const redisSub = new Redis(connectionObj)
```